test/test_chunkstore.py

```diff
@@ -191,6 +189,22 @@ def test_tiles_to_time_ranges(self):
                          [(tr[0].isoformat(), tr[1].isoformat()) for tr in time_ranges])
 
 
+class SentinelHubStore3DTestWithAllBands(SentinelHubStoreTest):
+    def get_cube_config(self):
+        return CubeConfig(dataset_name='S2L2A',
+                          geometry=(10.2, 53.5, 10.3, 53.6),
```

Comment:
Would you explicitly set band_names to None here?

Reply:
That is redundant because `band_names` is optional _and_ documented.